README.md

```diff
@@ -7,15 +7,20 @@ and associated libraries on the host, validating their functionality
 
 ## Build
 ```
-docker build -f Dockerfile  -t docker.io/alexeldeib/aks-gpu:latest .
+docker build -f Dockerfile  --build-arg DRIVER_VERSION=??? -t docker.io/alexeldeib/aks-gpu:latest .
 docker push docker.io/alexeldeib/aks-gpu:latest
-```
+``` 
+
+#### DRIVER_VERSION has to be one of:
+- 470.82.01
+- 510.47.03
+- 515.65.01
```

Comment:
This is not actually accurate, if you change it to "these versions are known to work" lgtm

I can clarify this offline, it really is just a matter of what's available from the various urls/sources